src/languages/en-US.ts

```diff
@@ -2511,15 +2511,17 @@ export default class extends Language {
 		COMMAND_QUOTE_EXTENDED: {},
 		COMMAND_ROLES_DESCRIPTION: 'List all public roles from a guild, or claim/unclaim them.',
 		COMMAND_ROLES_EXTENDED: {
-			extendedHelp: `Public roles? They are roles that are available for everyone, an administrator must configure
-					them with a configuration command.`,
+			extendedHelp: `Public roles? They are roles that are available for everyone, an administrator must configure them with a configuration command.`,
 			explainedUsage: [['Roles', 'The list of roles to claim and unclaim. Leave this empty to get a list of roles']],
 			reminder: [
 				'When using claim/unclaim, the roles can be individual, or multiple.',
 				'To claim multiple roles, you must separate them by a comma.',
-				'You can specify which roles by writting their ID, name, or a section of the name.'
+				'You can specify which roles by writting their ID, name, or a section of the name.',
+				'',
+				'Administrators can add public roles using `conf set roles.public <Role>`.'
```

Comment:
I have seen people trying `Skyra conf set roles.public <Free Games>` thinking that the `<>` were required for Skyra to know the input, a placeholder name feels more effective:
```suggestion
				'Administrators can add public roles using `Skyra conf set roles.public MyPublicRole`.'
```